ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/fieldTemplates/label.html

```diff
@@ -1,3 +1,3 @@
 <template data-sly-template.label="${@ componentId, labelValue, labelVisible, bemBlock}">
-  <label id="${componentId}-label" for="${componentId}" class="${bemBlock}__label" data-sly-test="${labelValue && labelVisible}">${labelValue}</label>
+  <label id="${componentId}-label" for="${componentId}" class="${bemBlock}__label" data-sly-test="${labelValue && (labelVisible==null || labelVisible==true)}">${labelValue}</label>
```

Comment:
if label is provided but hidden, then we should get the markup and hide it from client.

Reply:
Hi @nit23uec , as per our discussion, this needs to be fixed in case of label and short and long description . There is a separate ticket for it, that's why didn't handle it for now